.redocly.yaml

```diff
@@ -0,0 +1,9 @@
+apis:
+  main:
+    root: docs/api/apiv3/openapi-spec.yml
+
+lint:
+  extends:
+    - recommended
+  rules:
+    no-ambiguous-paths: off
```

Comment:
I tried the vscode plugin for redocly and it gave a better experience than RubyMine, so I sticked with it, and it needed a `.redocly.yaml` file. When absent, redocly outputs this:

```
No configurations were defined in extends -- using built in recommended configuration by default.
Warning! This default behavior is going to be deprecated soon.
```

So I think it is compatible with it. Maybe I should have put my exceptions in that file.

I agree with ambiguous paths being bad design. But wanted to get rid of the inaccurate warnings in the openapi yaml file:

```yaml
  "/api/v3/queries/columns/{id}":
    "$ref": "./paths/queries_column.yml"
  "/api/v3/queries/{id}/star": # Paths should resolve unambiguously. Found two ambiguous paths: `/api/v3/queries/columns/{id}` and `/api/v3/queries/{id}/star`
    "$ref": "./paths/query_star.yml"
```

It is not smart enough to detect that `id` can only be integer and thus it will never be ambiguous.

I'd love to find a programmatic way to make it smarter about those ambiguous paths.

Reply:
Well, the path is still ambiguous in your example, even if `id` is an integer. Problem here is, that this can only be evaluated AFTER path resolvement (there are a lot of frameworks, that actually do not work like rails with grape here and already fail, when trying to build a backend with those pathes).

Reply:
ok, I'll remove the rule exclusion then.